uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/partials/_custom-datepicker.scss

```diff
@@ -0,0 +1,11 @@
+.small-datepicker {
+  width: 150px !important;
+  display: inline-block !important;
+  border-width: 0px !important;
+  -webkit-box-shadow: none !important;
+  -moz-box-shadow: none !important;
+}
+
+.small-datepicker:focus,.small-datepicker:active {
```

Comment:
//Format you styles

`
.small-datepicker {
   &:focus {
       ...
   }

   &:active {
      ...
   }
`